Multisig/Cross-layer/Configuration/Config.Example.xcconfig

```diff
@@ -186,7 +186,10 @@ WEB3AUTH_GOOGLE_CLIENT_ID = YOUR_WEB3AUTH_GOOGLE_CLIENT_ID
 WEB3AUTH_GOOGLE_VERIFIER = YOUR_WEB3AUTH_GOOGLE_VERIFIER
 // Use value from ios OAuth client. Credentials -> Safe{Wallet} (type ios) ->  Additional Information -> iOS URL scheme
 // Do not add :/$()/ at the end. It won't pass the build stage
-WEB3AUTH_REDIRECT_SCHEME = YOUR_WEB3AUTH_REDIRECT_SCHEME
+_WEB3AUTH_REDIRECT_URL_DEV = https:/$()/safe-wallet-web.dev.5afe.dev/web3auth/
```

Comment:
By doing this we don't use the value from Jenkins file is it intended ?

Reply:
The redirect urls are not secret. We do not need to take them from jenkins.